stm32cube/stm32h7xx/README

```diff
@@ -69,8 +65,8 @@ Patch List:
      ST Internal Reference: 136825
 
    *Fix to use transmit timestamp when it is valid
-    Read timestamp only when status flag is set in Tx descriptor.
```

Comment:
if related to ST  Internal Reference: 157129 it is not yet fixed (see ST Internal Reference: 157128)